__tests__/app.test.js

```diff
@@ -92,15 +92,174 @@ describe("testing app /api/articles", () => {
+    describe("Sort queries defaulted to desc", () => {
+      test("should return an array of article objects, status 200 and be sorted by title", () => {
+        return request(app)
+          .get("/api/articles?sort_by=title")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].title).toEqual("Z");
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by topic", () => {
+        return request(app)
+          .get("/api/articles?sort_by=topic")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].topic).toEqual("mitch");
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by author", () => {
+        return request(app)
+          .get("/api/articles?sort_by=author")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].author).toEqual("rogersop");
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by body", () => {
+        const expectedBody = "some gifs";
+        return request(app)
+          .get("/api/articles?sort_by=body")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].body).toEqual(expectedBody);
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by date when specified", () => {
+        return request(app)
+          .get("/api/articles?sort_by=date")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].created_at).toEqual("2020-11-03T09:12:00.000Z");
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by votes", () => {
+        return request(app)
+          .get("/api/articles?sort_by=votes")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].votes).toEqual(100);
+          });
+      });
+      test("should return an array of article objects, status 200 and be sorted by article_id", () => {
+        return request(app)
+          .get("/api/articles?sort_by=article_id")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].votes).toEqual(0);
+          });
+      });
+    });
+    describe("Order queries", () => {
+      test("should respond with array of articles in ascending order ", () => {
+        return request(app)
+          .get("/api/articles?order=asc")
+          .expect(200)
+          .then(({ body }) => {
+            const articles = body;
+            expect(articles[0].created_at).toEqual("2020-01-07T14:08:00.000Z");
+          });
```

Comment:
Is there a way we could make this more fluid? Perhaps jest toBeSortedBy would test the behaviour we want.